src/components/field/svg/FieldOverlayRoot.tsx

```diff
@@ -242,6 +242,17 @@ class FieldOverlayRoot extends Component<Props, State> {
                 <FieldImage2024 />
               </>
             )}
+
+            {layers[ViewLayers.FieldOverlays] &&
+              doc.pathlist.activePath.params.constraints.map((c) => {
```

Comment:
Can this be just the enabled constraints, and additionally filter out the one that's selected (if any)?